munki_rebrand.py

```diff
@@ -326,7 +326,13 @@ def convert_to_icns(png, output_dir, actool=""):
     # Munki 3.6+ has an Assets.car which is compiled from the Assets.xcassets
     # to provide the AppIcon
     if actool:
-        shutil.copytree("./Assets.xcassets/", xcassets, dirs_exist_ok=True)
+        # Use context of the location of munki_rebrand.py to find the Assets.xcassets directory.
+        rebrand_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), os.pardir))
+        xc_assets_dir = os.path.join(rebrand_dir, 'Assets.xcassets/')
+        if not os.path.isdir(xc_assets_dir):
+            print(f"The Assets.xcassets folder could not be found in {rebrand_dir}.\nMake sure it's in place, and then try again.")
```

Comment:
Could you stick with the %s method of string formatting for consistency with the rest of the script, please?